sqrl-server/sqrl-server-vertx-base/src/main/java/com/datasqrl/graphql/config/ServerConfig.java

```diff
@@ -47,29 +47,32 @@ public class ServerConfig {
   SwaggerConfig swaggerConfig;
   KafkaConfig.KafkaMutationConfig kafkaMutationConfig;
   KafkaConfig.KafkaSubscriptionConfig kafkaSubscriptionConfig;
+  JdbcConfig duckDbConfig;
+  JdbcConfig snowflakeConfig;
 
   public ServerConfig(JsonObject json) {
```

Comment:
This parser is so complicated, so complicated that I wonder if we should dump it write generic java records and use mapstrut to translate our objects into vertx objects.

This parser feels like 2004 hibernate